buildSrc/src/main/kotlin/org/apache/beam/gradle/BeamDependencies.kt

```diff
@@ -0,0 +1,253 @@
+        const val slf4j_jdk14                                 = "org.slf4j:slf4j-jdk14:$slf4j_version"
+        const val slf4j_log4j12                               = "org.slf4j:slf4j-log4j12:$slf4j_version"
+        const val snappy_java                                 = "org.xerial.snappy:snappy-java:1.1.4"
+        const val spark_core                                  = "org.apache.spark:spark-core_2.11:$spark_version"
+        const val spark_network_common                        = "org.apache.spark:spark-network-common_2.11:$spark_version"
+        const val spark_sql                                   = "org.apache.spark:spark-sql_2.11:$spark_version"
+        const val spark_streaming                             = "org.apache.spark:spark-streaming_2.11:$spark_version"
+        const val stax2_api                                   = "org.codehaus.woodstox:stax2-api:4.2.1"
+        const val testcontainers_clickhouse                   = "org.testcontainers:clickhouse:$testcontainers_version"
+        const val testcontainers_elasticsearch                = "org.testcontainers:elasticsearch:$testcontainers_version"
+        const val testcontainers_kafka                        = "org.testcontainers:kafka:$testcontainers_version"
+        const val testcontainers_localstack                   = "org.testcontainers:localstack:$testcontainers_version"
+        const val testcontainers_postgresql                   = "org.testcontainers:postgresql:$testcontainers_version"
+        const val testcontainers_gcloud                       = "org.testcontainers:gcloud:$testcontainers_version"
+        const val vendored_bytebuddy_1_10_8                   = "org.apache.beam:beam-vendor-bytebuddy-1_10_8:0.1"
+        const val vendored_grpc_1_26_0                        = "org.apache.beam:beam-vendor-grpc-1_26_0:0.3"
+        const val vendored_guava_26_0_jre                     = "org.apache.beam:beam-vendor-guava-26_0-jre:0.1"
+        const val vendored_calcite_1_20_0                     = "org.apache.beam:beam-vendor-calcite-1_20_0:0.1"
+        const val woodstox_core_asl                           = "org.codehaus.woodstox:woodstox-core-asl:4.4.1"
+        const val zstd_jni                                    = "com.github.luben:zstd-jni:1.4.5-2"
+        const val quickcheck_core                             = "com.pholser:junit-quickcheck-core:$quickcheck_version"
+    }
+    object groovy {
+        const val groovy_all = "org.codehaus.groovy:groovy-all:2.4.13"
+    }
+    object maven {
+        const val maven_compiler_plugin = "maven-plugins:maven-compiler-plugin:3.7.0"
+        const val maven_exec_plugin     = "maven-plugins:maven-exec-plugin:1.6.0"
+        const val maven_jar_plugin      = "maven-plugins:maven-jar-plugin:3.0.2"
+        const val maven_shade_plugin    = "maven-plugins:maven-shade-plugin:3.1.0"
+        const val maven_surefire_plugin = "maven-plugins:maven-surefire-plugin:3.0.0-M5"
+    }
```

Comment:
Is it possible to also remove `libraries` from `BeamModulePlugin.groovy`?

If the only way to do this is to duplicate the definitions, I don't think we should. These dependencies change quite often so the two definitions would start diverging.

Reply:
Right now the only way is to duplicate the definitions.
I guess we should move dependencies declaration to Kotlin at the same time as moving `BeamModulePlugin`

Reply:
ok thanks, sorry about that.